component/pom.xml

```diff
@@ -86,6 +95,11 @@
             <artifactId>siddhi-store-rdbms</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.siddhi.extension.store.mongodb</groupId>
+            <artifactId>siddhi-store-mongodb</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Also check all the plugins used in the pom and see whether you need them

Reply:
Required to run MongoDB tests written.